var/spack/repos/builtin/packages/samrai/package.py

```diff
@@ -66,11 +66,10 @@ class Samrai(AutotoolsPackage):
     depends_on('hdf5+mpi')
     depends_on('m4', type='build')
 
-    # At some point later versions of boost were not able to be found
-    # by SAMRAI during configure, so we're using boost <= 1.60.0 for
-    # < 3.12.0 versions. I don't know what version of boost that
-    # happened at though without trying each version of boost.
-    depends_on('boost@:1.60.0', when='@0:3.11.99')
+    # From 3.12.0, samrai no long depends on boost.
+    # version 3.11.5 or earlier can only work with boost version
+    # 1.64.0 or earlier.
+    depends_on('boost@:1.64.0', when='@0:3.11.99', type='build')
```

Comment:
It doesn't link to any of the Boost libraries?

Reply:
Yes.